lib/matplotlib/projections/polar.py

```diff
@@ -1182,12 +1183,87 @@ def set_rorigin(self, rorigin):
     def get_rorigin(self):
         return self._originViewLim.y0
 
-    def set_rlim(self, *args, **kwargs):
+    def set_rlim(self, bottom=None, top=None, emit=True, auto=False, **kwargs):
+        """
+        See `~.polar.PolarAxes.set_ylim`
```

Comment:
See `~.polar.PolarAxes.set_ylim`. Additionally, you can pass *rmin*/*rmax* in place of *ymin*/*ymax*.

Reply:
?? Not following this.  We are trying to deprecate rmin, ymin etc.  (Well, I don't agree with calling them bottom and top, but...)

Reply:
Thought to document the current state, even if it's deprecated (well, which should be mentioned then). But since this had no documentation at all so far. It's also ok to keep this a secret.